pkg/deploy/generator/resources_oic.go

```diff
@@ -23,15 +23,15 @@ var (
 
 func (g *generator) oicStorageAccount() *arm.Resource {
 	storageAccount := &mgmtstorage.Account{
-		Kind: mgmtstorage.StorageV2,
+		Kind: mgmtstorage.KindStorageV2,
 		Sku: &mgmtstorage.Sku{
 			Name: "Standard_LRS",
 		},
 		AccountProperties: &mgmtstorage.AccountProperties{
```

Comment:
Are we scoping the OIC storage accounts too for this story?
If yes, then we need to disable the SAS keys here too.
The OIC storage accounts will be accessed via EntraID and will not need the SAS keys, thus can be disabled.
The INT and PROD pipelines for deploying the OIC storage accounts will also need a change for disabling the SAS keys.

Reply:
Hi @rajdeepc2792 , to the best of my knowledge, this story doesn't have OIC in scope.
At least it didn't, at the time I got it with @nwnt .
The reason why I changed from `StorageV2` to `KindStorageV2` here was to make it compile.
The `mgmt/storage` package was upgraded to a different version for the whole repo as a pre-requisite for RP and Gateway, and there was a usage of it there in OIC that I had to change as part of this upgrade.
So it's more of a coincidence, not a SAS change that was left behind for this story.

Please correct me if I'm wrong @nwnt . Thank you!